app/src/pages/inside/settingsPage/settingsTabs/notificationsTab/notificationsTab.jsx

```diff
@@ -19,33 +19,34 @@ const cx = classNames.bind(styles);
   (state) => ({
     projectRole: activeProjectRoleSelector(state),
     userRole: userAccountRoleSelector(state),
-    emailEnabled: projectEmailEnabledSelector(state),
-    emailCases: projectEmailCasesSelector(state),
+    enabled: notificationIntegrationSelector(state).enabled,
```

Comment:
Please add corresponding selector to incapsulate name and position of this property.